test/helpers/cilium.go

```diff
@@ -671,19 +686,70 @@ func (s *SSHMeta) ServiceDelAll() *CmdRes {
 
 // SetUpCilium sets up Cilium as a systemd service with a hardcoded set of options. It
 // returns an error if any of the operations needed to start Cilium fails.
-func (s *SSHMeta) SetUpCilium() error {
-	template := `
-PATH=/usr/lib/llvm-3.8/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
-CILIUM_OPTS=--kvstore consul --kvstore-opt consul.address=127.0.0.1:8500 --debug --debug-verbose flow
-INITSYSTEM=SYSTEMD`
+func (s *SSHMeta) SetUpCilium(ciliumOpts *CiliumOpts) error {
+	if ciliumOpts == nil {
+		ciliumOpts = &CiliumOpts{
+			KVStore:      "consul",
+			KVStoreOpt:   "consul.address=127.0.0.1:8500",
+			Debug:        true,
+			DebugVerbose: "flow",
+		}
+	}
+
+	template := `PATH=/usr/lib/llvm-3.8/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
+CILIUM_OPTS=--kvstore={{.KVStore}} --kvstore-opt={{.KVStoreOpt}} --debug={{.Debug}} --debug-verbose={{.DebugVerbose}}
+INITSYSTEM=SYSTEMD
+`
+
+	err := RenderTemplateToFile("cilium-opts", template, ciliumOpts, 0644)
+	if err != nil {
+		return err
+	}
+	defer os.Remove("cilium-opts")
+	ciliumOptsPath := filepath.Join(s.ciliumRootPath, "test", "cilium-opts")
+
+	res := s.Exec(fmt.Sprintf("sudo cp %s /etc/sysconfig/cilium", ciliumOptsPath))
+	if !res.WasSuccessful() {
+		return fmt.Errorf("%s", res.CombineOutput())
+	}
+	res = s.Exec("sudo systemctl restart cilium")
+	if !res.WasSuccessful() {
+		return fmt.Errorf("%s", res.CombineOutput())
+	}
+	return nil
+}
+
+// SetUpCilium sets up Cilium as a systemd service with a hardcoded set of options. It
```

Comment:
Improper documentation. Specify on which environments this is supported as well.